### DISCUSSION TITLE
warning when using MultiAppGeneralFieldNearestLocationTransfer from element to nodes

### DISCUSSION THREAD

**Discussion by mcacace** [idaholab/moose#26093] [Q&A General]
Dear all,
I'm trying to map an elemental auxvariables (subapp) to a nodal auxvariable (master app) but keep receiveing the following warning - multiple centroids (see below). 

_"Transfer is projecting from nearest-centroids to nodes. This is likely causing floating point indetermination in the results because multiple centroids are 'nearest' to a node. Please consider using a ProjectionAux to build a nodal source variable (for example linear Lagrange) before transferring"_

However, when I use the ProjectionAux inside the master app to do the mapping the nodal variable to which I map returns zeroes. I tried to force the computation of the elemental Auxvariables during nonlinear and linear iteration while having the projection only at timestep_end but I keep receiving zeroes. 
Does anone have any idea how to best project an elemental auxvariable to a nodal auxvariable? 
Thanks,
Mauro

**Comment by GiudGiud** [comment]
Hello

I added this warning. 
ProjectionAux should have been the way.
Can you project in the subapp to a nodal variable instead?

Guillaume
